docs/user/reference/metrics/timing_distribution.md

```diff
@@ -239,6 +239,101 @@ Glean.pages.pageLoad.stopAndAccumulate(timerId);
 
 {{#include ../../../shared/tab_footer.md}}
 
+### `accumulateSamples`
+
+Accumulates the provided signed samples in the metric.
+This is required so that the platform-specific code can provide us with
+64 bit signed integers if no `u64` comparable type is available. This
+will take care of filtering and reporting errors for any provided negative
+sample.
+
+Please note that this assumes that the provided samples are already in
+the "unit" declared by the instance of the metric type (e.g. if the
+instance this method was called on is using `TimeUnit::Second`, then
+`samples` are assumed to be in that unit).
+
+This discards any negative value in `samples` and report an `ErrorType::InvalidValue`
+for each of them. Reports an `ErrorType::InvalidOverflow` error for samples that
+are longer than maximum sample time for the given unit.
+
+{{#include ../../../shared/tab_header.md}}
+
+<div data-lang="Kotlin" class="tab">
+
+```Kotlin
+import org.mozilla.yourApplication.GleanMetrics.Pages
+
+fun onPageLoaded(e: Event) {
+    Pages.pageLoad.accumulateSamples(samples)
+}
+```
+
+</div>
+<div data-lang="Java" class="tab">
+
+```Java
+import org.mozilla.yourApplication.GleanMetrics.Pages;
+
+void onPageLoaded(Event e) {
+    Pages.INSTANCE.pageLoad().accumulateSamples(samples);
+}
+```
+
+</div>
+<div data-lang="Swift" class="tab">
+
+```Swift
+import Glean
+
+func onPageLoaded() {
+    Pages.pageLoad.accumulateSamples(samples)
+}
+```
+
+</div>
+<div data-lang="Python" class="tab">
+
+```Python
+from glean import load_metrics
+metrics = load_metrics("metrics.yaml")
+
+class PageHandler:
+    def on_page_loaded(self, event):
+        metrics.pages.page_load.accumulate_samples(samples)
+```
+
+</div>
+<div data-lang="Rust" class="tab">
+
+```Rust
+use glean_metrics::pages;
+
+fn on_page_loaded() {
+    pages::page_load.accumulate_samples(samples);
```

Comment:
Might not work. API isn't on [the trait](https://searchfox.org/glean/source/glean-core/src/traits/timing_distribution.rs) and I _think_ `glean_metrics::pages`' type is only advertised as adhering to the trait? (Certainly doesn't work on FOG, so it at least needs a disclaimer)

Reply:
You are correct, I had added the RLB version without looking at the trait, before getting to the FOG versions and asking about it.  I'll remove this and add a link to the same bug as the FOG implementations